insights: added changelog note for missing internal actor (#29756)
sourcegraph/sourcegraph: CHANGELOG.md

- Issue preventing searches from completing when certain patterns contain `@`. [#29489](https://github.com/sourcegraph/sourcegraph/pull/29489)
- The grafana dashboard for "successful search request duration" reports the time for streaming search which is used by the browser. Previously it reported the GraphQL time which the browser no longer uses. [#29625](https://github.com/sourcegraph/sourcegraph/pull/29625)

> - A regression introduced in 3.35 causing Code Insights that are run over all repositories to not query against repositories that have permissions enabled. These restricted repositories remain filtered when a user views a chart based on their user permissions. This may cause global Insights to undercount for data points generated after upgrading to 3.35. [](https://github.com/sourcegraph/sourcegraph/pull/29725)

Reviewer: ```suggestion
- A regression introduced in 3.35 causing Code Insights that are run over all repositories to not query against repositories that have permissions enabled. (Restricted repositories are and remain filtered based on user permissions when a user views a chart, not at query time.) This caused global Insights to undercount for data points generated after upgrading to 3.35. [](https://github.com/sourcegraph/sourcegraph/pull/29725)
```
